src/appleseed.studio/CMakeLists.txt

```diff
@@ -288,6 +290,8 @@ set (mainwindow_sources
     mainwindow/configurationmanagerwindow.cpp
     mainwindow/configurationmanagerwindow.h
     mainwindow/configurationmanagerwindow.ui
+    mainwindow/droppedmaterialmenu.cpp

```

Comment:
Maybe we should rename these files to `MaterialDropMenu` to match `MaterialDropHandler` and make it clearer that these components are related. What do you think?

Reply:
DroppedMaterialMenu is no longer use, it was created to keep the mouse position and the material name when the menu is open but with the new handler, I no longer need it as the information are stocked in the handler